availability-store/src/lib.rs

```diff
@@ -50,6 +50,8 @@ use std::pin::Pin;
 
 mod worker;
 mod store;
+pub mod networking;
```

Comment:
> The goal would be to have fully self-contained logic for availability networking within its crate. 

Why expose the `networking` module to other crates then?

Reply:
Because you still need to expose the structs and types so you can use then in the networking crate.